### ISSUE TITLE
Retrospectives: Feedback card takes 5 or more tabs to get through one card, also non-actionable item is focused.

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vsts-extension-retrospectives#1050]
### Tags:
#A11y;  #accessibility;

### Repro steps:
1. Install the Retrospectives Extension:- Go to the Azure DevOps Marketplace and search for the "Retrospectives" extension.- Click "Get it free" and follow the prompts to install it to your Azure DevOps organization.
2. Create a Retrospective:

- Navigate to your Azure DevOps project.
- In the left-hand menu, select "Boards" and then "Retrospectives."
- Click on "New Retrospective" to create a new session.

3. Add some feedback items under each column
4. Navigate around to see if the keyboard navigation is standard and efficient.

### Actual result:

1. Non interactive card item is a tab stop (when it should not be).
2. Each card required multiple tab to get through it, very inefficient.

Note: (tracking in another bug) Tabs in a tab list cannot be arrow to, and each tab is a tab stop (non standard interaction.

### Expected Result
1. Non interactive card items should not receive the keyboard focus. 
2. Once we tab to any card then all the interactive control of that car should be accessible with arrow key and pressing TAB key user can move to next care.
 

### Attachment

![Image](https://github.com/user-attachments/assets/0435a1e2-6d6d-48c6-a997-7a0eb156fd86)

![Image](https://github.com/user-attachments/assets/5512a21b-bb3a-434c-b1d5-2fc5cd2eaaab)

**Comment by msftedad** [comment]
A11y issue
